src/components/app.js

```diff
@@ -18,11 +20,12 @@ export default class App extends Component {
 		}
 	}
 
-	render({ url }) {
+	render() {
+		let url = store.getState().url;
```

Comment:
Personally I like to get this in the HOC instead of drilling it down (nitty) but this maybe isn't the time and place to discuss this.

Reply:
Yeah I agree. It should be passed through `context` instead. 8.x doesn't have the createContext API and I think this will be a lot easier to do once we switch to X with hooks here 👍 I'm kinda postponing cleanups like that until the switch.

Reply:
Agreed